lib/services/local/presence.go

```diff
@@ -1500,8 +1500,8 @@ func (s *PresenceService) listResources(ctx context.Context, req proto.ListResou
 		keyPrefix = []string{nodesPrefix, req.Namespace}
 		unmarshalItemFunc = backendItemToServer(types.KindNode)
 	case types.KindWindowsDesktopService:
-		keyPrefix = []string{windowsDesktopServicesPrefix, req.Namespace}
-		unmarshalItemFunc = backendItemToServer(types.KindWindowsDesktopService)
+		keyPrefix = []string{windowsDesktopServicesPrefix}
+		unmarshalItemFunc = backendItemToWindowsDesktopService
```

Comment:
I'm still not familiar with this part of the codebase and am a bit confused by this change.

Could you explain me why it's ok to remove `req.Namespace` from the keyPrefix only in this case?

I'm also not sure what how this change is linked to the PR. If I understand correctly the WindowsDekstopService were not unmarshalled properly by `UnmarshalServer` and this is a fix?

Reply:
Thanks for the explanation :)